config/locales/en.yml

```diff
@@ -2900,6 +2912,23 @@ en:
       showing_page: "Page %{page}"
       next: "Next »"
       previous: "« Previous"
+  user_mutes:
+    index:
+      you_have_muted_n_users:
+        zero: "You haven't muted any Users 🎉"
+        one: "You have muted %{count} User"
+        other: "You have muted %{count} Users"
+      user_mute_explainer: "Private messages of muted users are moved into a separate inbox and you won't receive email notifications."
+      user_mute_admins_and_moderators: "You can mute Admins and Moderators but their private messages cannot."
+      unmute: "Unmute"
+      send_message: "Send message"
+      title: "Muted Users"
+    create:
+      notice: "You muted %{name}."
+      error: "Something went wrong."
+    destroy:
+      notice: "You unmuted %{name}."
+      error: "Something went wrong."
```

Comment:
Either use controller action specific messages for the errors or reuse generic messages.